docs/index.md

```diff
@@ -4090,11 +4090,14 @@ Here are some conventions for the source code itself:
 6. Use the `->` (pipe) construct as is now conventional in ReScript.
    The `->` is syntactic sugar that allows you to call a function
    but with the parameter to the left of `->` becoming the first (or specified)
-   parameter of the function. For example, function call
-   `myData->MyModule.myFunction` means the same thing as
-   `MyModule.myFunction(myData)`, and function call
-   `myData->MyModule.myFunction(fooData)` means the same thing as
-   `MyModule.myfunction(myData, fooData)`.
+   parameter of the function. For example:
+
+   | Expression | Means |
+   | ---------- | ----- |
+   | <tt>myData->MyModule.myFunction</tt> | <tt>MyModule.myFunction(myData)</tt> |
+   | <tt>myData&#45;></tt>&ZeroWidthSpace;<tt>MyModule.myFunction(fooData)</tt> | <tt>MyModule.myfunction(</tt>&ZeroWidthSpace;<tt>myData, fooData)</tt> |
+   | <tt>myData&#45;></tt>&ZeroWidthSpace;<tt>MyModule.myFunction(fooData, _)</tt> | <tt>MyModule.myfunction(</tt>&ZeroWidthSpace;<tt>fooData, myData)</t> |
+
```

Comment:
Please add the below remark (please feel free to rephrase it if it contains grammatical errors or sounds awkward):

```
Please note, that usage of the pipe construct is not obligatory if you want to contribute to the codebase. 
If you prefer, you may still use conventional function application like func(arg1,arg2). 
However, during the PR review, you may be asked to use the pipe construct in some places, 
especially in cases like func1(func2(func3(arg)))
```